src/app/delegation_verify/submission.ml

```diff
@@ -81,7 +81,7 @@ module Filesystem = struct
     { submitter
     ; snark_work
     ; block_hash = meta.block_hash
-    ; submitted_at = meta.submitted_at
+    ; submitted_at = meta.created_at
```

Comment:
Hm, I believe it was like that before and we explicitly changed it to `submitted_at`... The reason was that we had some duplicated/incomplete rows in cassandra (that's because submitted_at and created_at are not always the same and submitted_at field is part of compound key in the `submissions` table)... so was there any rationale for making this change now? I fear it may cause as problems as we had before...

Reply:
The rationale is that the statetless-verifier has an argument that allows it to read submissions from the file-system. However, the submission doesn't have a structure of this type, i.e. the blocks don't have submitted_at as a field (at least not the blocks I was looking at). It needed this change to read blocks from the file system.

Reply:
Indeed, the submission does not have a field `submitted_at` inside the file, however it is part of the file name (`<submitted_at_date>/<submitted_at>-<submitter>.json`) so it is available. I think I'd be for reverting it back and maybe parse the filename in case of `fs` mode? I think that `submitted_at = meta.created_at` may cause some inconsistencies.

Reply:
There should be no impact on Cassandra. I tested this by the following and i just got the pickles error, i.e. no syntax error:
```
_build/default/src/app/delegation_verify/delegation_verify.exe \
cassandra \
--keyspace pm900 \
'2024-02-09 07:58:39.0+0000' \
'2024-02-09 07:58:48.0+0000'
```

Reply:
Ah, you're right!